src/fs_store.rs

```diff
@@ -39,8 +40,9 @@ use error::ErrorKind;
 /// on other peers and for the splaying to be useful we need to guarantee a
 /// uniform distribution of documents across the subdirectories.
 ///
-/// Likewise we use the hex encoding of the document ID as the filename to avoid
-/// any issues with non-UTF8 characters in the document ID.
+/// Likewise we use the hex encoding of the document ID as the 2nd-level
+/// directory name to avoid any issues with non-UTF8 characters in the document
```

Comment:
This is actually a lie as the implementation doesn't support non-UTF8 characters at all (lol)